spyder/preferences/general.py

```diff
@@ -25,6 +26,10 @@
 from spyder.preferences.configdialog import GeneralConfigPage
 from spyder.py3compat import to_text_string
 import spyder.utils.icon_manager as ima
+if sys.platform == "darwin":
+    import applaunchservices as als
+    from spyder.utils.qthelpers import (register_app_launchservices,
+                                        restore_launchservices)
```

Comment:
Now you can import these functions from `qthelpers` normally and not inside the `if sys.platform == "darwin":` line above.